src/lib/Makefile.am

```diff
@@ -94,7 +106,8 @@ libreport_la_CPPFLAGS = \
     -D_GNU_SOURCE
 libreport_la_LDFLAGS = \
     -ltar \
-    -version-info 0:1:0
+    -version-info 0:1:0 \
```

Comment:
What do you think about using something like https://gitlab.gnome.org/GNOME/glib/-/blob/ac396b4dc14f53dda7c7263f3c4ae10c04e8b5d4/configure.ac#L41 instead?

Reply:
It seems like more complicated way if using the version info to me. Like the way they do odd and even versions for debugging. I am not sure what is would be the benefit of that for us.

Reply:
My point was more using macros and not hardcoding versions.

Reply:
Howver I miss in the patches LD version script, which, I added but somehow deleted with the po file changes (sigh). Here is info about version scripts. The .sym file is in format conusamable by LD version scripts.

Here is info about the LD scripts: https://www.gnu.org/software/gnulib/manual/html_node/LD-Version-Scripts.html

Reply:
From ernestask:
"My point was more using macros and not hardcoding versions."

Sorry, I completely missed that you wrote this. However, I do not kow what is the advantage of using such macros (there probably is some, otherwise people would not make them, but modifying 3 numbers looks easier to me and in general I would prefer to avoid creating more complicated logic that way, because at least for me this macro language is a bit hard to debug).